### PR TITLE
add rotate transforms for image

### PR DESCRIPTION
**Pull request by knorth55**
add `rotate` for `img`.
`random_rotate` exists but `rotate` does not exists.

### CODE REVIEW

File: tests/transforms_tests/image_tests/test_rotate.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from chainercv.transforms import rotate
+
+
+@testing.parameterize(*testing.product({
+    'k': [0, 1, 2, 3]
+}))
+class TestRotate(unittest.TestCase):
+
+    def test_rotate(self):
+        img = np.random.uniform(size=(3, 24, 24))
```
Comment (Hakuyume) [inline tests/transforms_tests/image_tests/test_rotate.py]: How about using non-square image for test?

Comment (knorth55) [reply tests/transforms_tests/image_tests/test_rotate.py]: updated

**Comment by yuyu2172** [comment]
Could you change argument `k` to `angle`?
If possible, could you support continuous values between `-180,180`?

**Comment by knorth55** [comment]
i think continuous rotate support is different problem.
continuous rotate do not change `img.shape`, intepolate pixel value and do zero-padding in outside area.
this function is `rotate90` introduced in `random_rotate`.

**Comment by knorth55** [comment]
Updated to use `scipy.ndimage.rotate`

File: chainercv/transforms/image/rotate.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            '$ pip install scipy')
+
+
+def rotate(img, angle, expand=True):
+    """Rotate images by degrees.
+
+    Args:
+        img (~numpy.ndarray): An arrays that get flipped. This is in
+            CHW format.
+        angle (float): Clock-wise rotation angle (degree) in [-180, 180].
```
Comment (yuyu2172) [inline chainercv/transforms/image/rotate.py]: https://github.com/chainer/chainercv/pull/692/files#r218293211

**Pull request merged**